src/components/icon/assets/logo_opensearch.js

```diff
@@ -11,29 +11,27 @@
 
 import * as React from 'react';
 
-const OuiIconLogoOpenSearch = ({ title, titleId, ...props }) => (
+const OuiIconLogoOpensearch = ({ title, titleId, ...props }) => (
   <svg
-    width={32}
-    height={32}
-    fill="none"
```

Comment:
The width and height were removed because they are missing in the original icon. Can you update the accompanying svg tag with the width and height?

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 64 64" fill="none">
```

Reply:
Thanks for the comment, it's done!

<img width="600" alt="Screenshot 2023-03-18 at 11 19 37" src="https://user-images.githubusercontent.com/17855243/226087149-e780505c-2a94-4a6f-8b2e-b4b1fe5e587c.png">